config.toml

```diff
@@ -0,0 +1,24 @@
+baseURL = ""  # This is populated by the github deploy workflow and is equal to "<domainName>/<version|latest>"
+languageCode = "en-us"
+title = "Apache Iceberg"
+theme= "hugo-book"
+
+[params]
+  BookLogo = "img/iceberg-logo-icon.png"
+  description = "The open table format for analytic datasets."
+  versions.iceberg = "" # This is populated by the github deploy workflow and is equal to the branch name
+  versions.nessie = "0.15.1"
+  latestVersions.iceberg = "0.12.1"  # This is used for the version badge on the "latest" site version
+
+[[params.social]]
+	title = "devlist"
+	icon = "envelope-o"
+	url = "mailto:dev-subscribe@iceberg.apache.org"
```

Comment:
Rather than linking to the dev list, what about linking to the community page that has a lot more information on the various ways to get involved?

Reply:
Good idea, updated it to link to the Mailing List portion of the community page and updated the title to "Mailing Lists"

Reply:
Why not just a "Community" link?